udata/harvest/backends/base.py

```diff
@@ -22,10 +23,32 @@
 requests.packages.urllib3.disable_warnings()
 
 
-MAPPABLE = (
-    'tags',
-    'license',
-)
+class HarvestFilter(object):
+    TYPES = {
+        str: 'string',
+        basestring: 'string',
+        int: 'integer',
+        bool: 'boolean',
+        UUID: 'uuid',
+        datetime: 'date-time',
```

Comment:
Is `date-time` a known convention? I would have written `datetime`, but not important.